build-system/babel-plugins/babel-plugin-transform-json-configuration/test/fixtures/transform/transform-internal-runtime-version-call/input.js

```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+jsonConfiguration({});
+
+jsonConfiguration({a: 1});
+
+jsonConfiguration({'b': ['test']});
+
+jsonConfiguration({
+  // comment
+  c: true,
+  d: {
+    e: false,
+  }
+});
```

Comment:
Can we include a couple illegal test cases here? E.g. string template, `+` operator etc.

Reply:
The only thing that will actually fail to transform is when the configuration references local variables, which the transformer won't have in scope during the evaluation. Anything statically computable will actually work fine, it's just the linter that forbids them.